test_communication/test/subscriber_py.py

```diff
@@ -76,9 +76,11 @@ def listener(message_name):
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument('message_name', default='Primitives',
                         help='name of the ROS message')
+    parser.add_argument('namespace', default='',
```

Comment:
since this isn't optional a default doesn't really make sense (for `message_name` too, but it started out being optional FWICT and morphed over time)

Reply:
Yeah, I see it as a convenience as I can test the "default" configuration by hand without have to type much and can test more "advanced" cases by typing a bit more. But I guess now that we have ros2 topic pub and echo, this convenience could be removed from these tests. I dont feel strongly about it.

Reply:
can you though, with these settings? I get:
```
python3 test_communication/test/subscriber_py.py
usage: subscriber_py.py [-h] message_name namespace
subscriber_py.py: error: the following arguments are required: message_name, namespace
```
I think it _was_ possible when this was first written but not anymore

Reply:
yeah right given that they are both positional they can't have defaults anymore (sorry 2AM here)

Reply:
addressed in https://github.com/ros2/system_tests/pull/256/commits/293db5e1008f265aeffb900267dcb6f8ccd257bc and rebased / resolved conflicts